spec/search-api.yaml

```diff
@@ -89,6 +90,11 @@ paths:
         '500':
           description: An unexpected error occurred
 components:
+  securitySchemes:
+    bearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
```

Comment:
I don't think we actually want to make any claims/restrictions on the format of the token do we?

Reply:
Good point - I think that's fair, although we'll likely need JWT to be DURI-friendly in the future.